### DISCUSSION TITLE
Site blocking is not working, would anyone be kind enough to help me figure out why?

### DISCUSSION THREAD

**Discussion by Retrofire47** [proginosko/LeechBlockNG#482] [Q&A]
i've been fiddling with LeechBlock for the last hour or 2. The design of this extension sounds ingenious and life-changing for me, but i only got it to work once, and it promptly stopped working again. I'm sure that i'm doing something stupid. More than happy to provide any additional information that might help

i am trying to block sites 24/7. so i have the "All Day" option selected [0000-2400], every day of the week, and "immediately block pages on these sites..".

**Comment by proginosko** [comment]
Please export your options to a text file (Options > General > Export Options) and email it to me at leechblockng@proginosko.com so I can take a closer look.
